packages/node/src/node/server/InteractionServer.ts

```diff
@@ -563,11 +540,10 @@ export class InteractionServer implements ProtocolHandler, InteractionRecipient
         }
 
         logger.debug(
```

Comment:
I think maybe this should log at INFO?

Reply:
We have the final subscription init as info ... no idea if we need the list as info ... normally not